backend/fms_core/viewsets/container.py

```diff
@@ -170,12 +170,13 @@ def search(self, _request):
         """
         Searches for parent containers that match the given query
         """
-        search_input = _request.GET.get("q")
+        search_input = _request.GET.get("q", None)
         is_parent = _request.GET.get("parent") == 'true'
         is_sample_holding = _request.GET.get("sample_holding") == 'true'
         is_exact_match = _request.GET.get("exact_match") == 'true'
+        except_kinds = _request.GET.get("except_kinds").split(",") if _request.GET.get("except_kinds") else []
```

Comment:
Using `or` would reduce repetition. ;)

Reply:
It would cause an error since None does not have a split() method ... i could store the initial result not sure what is really better ... I'll break it down to save on repeated calls.

Reply:
Oh woops. Didn't notice the split.